doc/nrf/getting_started/installing.rst

```diff
@@ -343,6 +342,14 @@ This is a simplified structure preview.
 There are additional folders, and the structure might change over time.
 The full set of repositories and folders is defined in the manifest file.
 
+In order to change the state of development you want to use to a different revision, enter the following commands from the :file:`nrf` directory you created earlier::
+
+  git checkout *next_revision*
+  west update
+
+Note that in this case, *next_revision* can be a SHA (for example, ``224bee9055d986fe2677149b8cbda0ff10650a6e``) in addition to a branch or tag name.
```

Comment:
```suggestion
In this case, *next_revision* can be a SHA (for example, ``224bee9055d986fe2677149b8cbda0ff10650a6e``) in addition to a branch or tag name.
```
Do you mean that it can be either a SHA, a branch name, or a tag name? Or that you can use SHA+branch name or SHA+tag name?

Reply:
Yes, in this case it can be any of those